src/refresh-token/refresh-token.controller.ts

```diff
@@ -10,6 +11,7 @@ export class RefreshTokenController {
   ) {}
 
   @Post()
+  @UseFilters(new ExceptionsFilter())
```

Comment:
Let's not use the Expection filter property in this way. We can define it globally only once in Main. 

`app.useGlobalFilters(new HttpExceptionFilter());`